### PR TITLE
Product design - hero, overview, key to success, case study sections

### PR DESCRIPTION
**Pull request by galdranorn**

### CODE REVIEW

File: _includes/services/faq.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ window.addEventListener('load', () => {
     const faqType = $faqContainer[0].dataset.faqType;
     const mobileDevice = $(window).width() < 811;
 
-    if ((faqType === 'standard' && mobileDevice) || faqType === 'custom') runFaq(e);
+    if ((faqType === 'standard' && mobileDevice) || faqType === 'custom') {
+      runFaq(e);
+    }
+
+    console.log('clicked')
```
Comment (siaradona) [inline _includes/services/faq.js:15]: .

File: _sass/new-styles/modules/services/_faq.scss [hunk 2]
```diff
@@ -37,22 +37,34 @@
...
         &--rotated {
           transform: rotate(90deg);
         }
       }
+
+      &__orange-arrow {
+        transform: rotate(90deg);
+        width: 20px;
+
+        &.services-faq__icon--rotated {
```
Comment (siaradona) [inline _sass/new-styles/modules/services/_faq.scss:64]: &--rotated (?)

File: _sass/new-styles/modules/services/product-design/product-design/_overview.scss [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+    &-image {
+      display: block;
+      width: auto;
+      height: 350px;
+      margin-bottom: -250px;
+    }
+
+    &-content, &-side {
+      box-sizing: border-box;
```
Comment (siaradona) [inline _sass/new-styles/modules/services/product-design/product-design/_overview.scss:96]: .

File: _sass/new-styles/modules/services/_faq.scss (hunk 2 above)
Comment (galdranorn) [reply _sass/new-styles/modules/services/_faq.scss:64]: as discussed, class not changed to not add additional case in js file

**Pull request merged**